posts/2015-05-24-es6-gotchas.md

```diff
@@ -46,6 +46,21 @@ export default class ComponentName extends React.Component {
   }
 ~~~
 
+Another alternative syntax is to use the `::` ES7 [bind](https://github.com/zenparsing/es-function-bind)
+
+~~~js
+export default class ComponentName extends React.Component {
+
+  handleClick(e) => {
+    ...
+  }
+  
+  render(){
+    return (
+      <button onClick={::this.handleClick}> click </button> 
```

Comment:
@tmbtech 

I would say something along the lines of:

>It may seem tempting to use `.bind()` or the ES7 `::` syntax to bind handlers in the component's `render()` method. However, ...